File: C/Arreglos/ARR_DEF_CONCEPT_2.c
#include <stdio.h>
#include <stdlib.h> 
#include <time.h> 

void CARGA_TECLADO_VECTOR(int v[10]) {
	int i;

	//printf("%p\n", v);

	for(i = 0; i < 10; ++i) {		
		//Genero numeros entre 0 y 49
		v[i] = rand() % 50;
		}
}

void CARGA_TECLADO_MATRIZ(int m[5][5]) {
	int i, j;

	for(i = 0; i < 5; ++i) {
		for(j = 0; j < 5; ++j) {
			//Genero numeros entre 0 y 49
			m[i][j] = rand() % 50;
			}
		}
}

int main() {	
	int i, j;
	int V[10];
	int M[5][5];

	// Uso el tiempo actual para el generador de numeros 
    srand(time(0)); 

	CARGA_TECLADO_VECTOR(V);

	CARGA_TECLADO_MATRIZ(M);

	printf(" ---------------------\n");
	printf(" ----- RESULTADOS ----\n");
	printf(" ---------------------\n");

	printf("Vector: \n");

	//valores de los elementos del vector
	for(i = 0; i < 10; ++i) {
		printf("%d ", V[i]);
	}

	printf("\n---------------------\n");
	printf("Matriz: \n");

	for(i = 0; i < 5; ++i) {
		for(j = 0; j < 5; ++j) {
			printf("%d ", M[i][j]);
		}
		printf("\n");
	}

	//getchar();

	return 0;
}

File: Python/Arreglos/ARR_DEF_CONCEPT_2.py
import random

def CARGA_TECLADO_VECTOR(v):
    for i in range(len(v)):
        v[i] = random.randint(0, 49)

def CARGA_TECLADO_MATRIZ(m):
    for i in range(len(m)):
        for j in range(len(m[i])):
            m[i][j] = random.randint(0, 49)

def main():
    random.seed()
    V = [0]*10
    M = [[0]*5 for _ in range(5)]
    CARGA_TECLADO_VECTOR(V)
    CARGA_TECLADO_MATRIZ(M)
    print(" ---------------------")
    print(" ----- RESULTADOS ----")
    print(" ---------------------")
    print("Vector: ")
    for val in V:
        print(val, end=' ')
    print()
    print("---------------------")
    print("Matriz: ")
    for fila in M:
        for val in fila:
            print(val, end=' ')
        print()
if __name__ == "__main__":
    main()